test/e2e/apimachinery/resource_quota.go

```diff
@@ -136,6 +137,95 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		framework.ExpectNoError(err)
 	})
 
+	/*
+		Release: v1.22
+		Testname: ResourceQuota, object count quota, service, allocateLoadBalancerNodePorts
+		Description: Create a ResourceQuota. Creation MUST be successful and its ResourceQuotaStatus MUST match to expected used and total allowed resource quota count within namespace.
+		Create a NodePort type Service. Its creation MUST be successful and resource usage count against the Service object and resourceQuota object MUST be captured in ResourceQuotaStatus of the ResourceQuota.
+		Create a LoadBalancer type Service that allocates node port. Its creation MUST be successful and resource usage count against the Service object and resourceQuota object MUST be captured in ResourceQuotaStatus of the ResourceQuota.
+		Create another LoadBalancer type Service that allocates node port. Its creation MUST fail as the node port resource quota is exceeded.
+		Create a LoadBalancer type Service that does not allocate node port. Its creation MUST be successful and resource usage count against the Service object and resourceQuota object MUST be captured in ResourceQuotaStatus of the ResourceQuota.
+		Create another LoadBalancer type Service that does not allocate node port. Its creation MUST fail as the loadbalancer resource quota is exceeded.
+		Create a ClusterIP type Service. Its creation MUST be successful and resource usage count against the Service object and resourceQuota object MUST be captured in ResourceQuotaStatus of the ResourceQuota.
+		Create another ClusterIP type Service. Its creation MUST fail as the service resource quota is exceeded.
+		Delete the Services. Deletion MUST succeed and resource usage count against the Service object MUST be released from ResourceQuotaStatus of the ResourceQuota.
+	*/
+	framework.ConformanceIt("should create a ResourceQuota and capture the life of a load balancer type service with allocateLoadBalancerNodePorts.", func() {
```

Comment:
start this as a regular e2e test... once the feature graduates to GA and this demonstrates stability, it can be promoted to a conformance test

Reply:
I see, just converted it to a e2e test, thanks!